### DISCUSSION TITLE
Give to users customize workflows

### DISCUSSION THREAD

**Discussion by eleventh-division** [community/community#36373] [Projects and Issues]
For example, I want to iterate WHEN branch created from issue SET status: In progress

**Comment by eleventh-division** [comment]
I want the board to be configurable so that I can work and not be distracted by dragging issues around the board

**Comment by ossanna16** [comment]
Hey there @eleventh-division and thank you for the feedback! I'm glad you found our community 🙂

**Comment by kksingla** [comment]
I want when the label is changed then the status & assignee of issue will be changed

**Comment by justinmchase** [comment]
I wanted `When: Added to Project -> Set: Milestone to X`

**Reply by justinmchase** [reply]
Or `Set: Label to Y`
